SpatialGDK/Source/SpatialGDKTests/SpatialGDKEditor/SpatialGDKEditorSchemaGenerator/SchemaGenObjectStub.h

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+#include "CoreMinimal.h"
+#include "SchemaGenObjectStub.generated.h"
+
+UCLASS()
+class USchemaGenObjectStub : public UObject
+{
+	GENERATED_BODY()
+public:
+
+	UPROPERTY(Replicated)
+	int IntValue;
+
+	UPROPERTY(Replicated)
+	bool BoolValue;
+};
+
+UCLASS(SpatialType)
+class USpatialTypeObjectStub : public UObject
+{
+	GENERATED_BODY()
+};
+
+UCLASS()
+class UChildOfSpatialTypeObjectStub : public USpatialTypeObjectStub
+{
+	GENERATED_BODY()
+};
+
+UCLASS(NotSpatialType)
+class UNotSpatialTypeObjectStub : public UObject
+{
+	GENERATED_BODY()
+};
+
+UCLASS()
+class UChildOfNotSpatialTypeObjectStub : public UNotSpatialTypeObjectStub
+{
+	GENERATED_BODY()
+};
+
+UCLASS()
+class UNoSpatialFlagsObjectStub : public UObject
+{
+	GENERATED_BODY()
+};
+
+UCLASS()
+class UChildOfNoSpatialFlagsObjectStub : public UNoSpatialFlagsObjectStub
+{
+	GENERATED_BODY()
+};
+
+UCLASS(SpatialType)
+class ASpatialTypeActor : public AActor
```

Comment:
`AChildOfSpatialTypeActor`?

Reply:
Why? It's a SpatialType class, that is also a child of AActor. So the name `ASpatialTypeActor` sounds about right in my opinion.

Reply:
Ah I meant, should we have a test for children of actors which are spatial type as well.